ax/utils/testing/torch_stubs.py

```diff
@@ -32,10 +31,26 @@ def get_torch_test_data(
                 [1.0 + offset, 2.0 + offset, 3.0 + offset],
                 [2.0 + offset, 3.0 + offset, 4.0 + offset],
             ],
+            # pyre-fixme[6]: For 2nd param expected `Optional[dtype]` but got
+            #  `Union[dtype, device]`.
+            # pyre-fixme[6]: For 2nd param expected `Union[None, str, device]` but
+            #  got `Union[dtype, device]`.
+            # pyre-fixme[6]: For 2nd param expected `bool` but got `Union[dtype,
+            #  device]`.
```

Comment:
Pyre doesn't understand **tkwargs.
```suggestion
            # pyre-ignore
```